src/gcodeExport.cpp

```diff
@@ -119,12 +120,16 @@ void GCodeExport::setInitialTemp(int extruder_nr, double temp)
 std::string GCodeExport::getFileHeader(const std::vector<bool>& extruder_is_used, const double* print_time, const std::vector<double>& filament_used, const std::vector<std::string>& mat_ids)
 {
     std::ostringstream prefix;
+
+    // output common headers
+    prefix << ";FLAVOR:" << toString(flavor) << new_line;
```

Comment:
You put this in front of the `;START_OF_HEADER` comment. Please revert this commit.